src/main/java/frc/robot/RobotContainer.java

```diff
@@ -97,6 +98,17 @@ private void configureBindings() {
     xboxController.povDown().whileTrue(new RunUptake(m_uptake, -0.5));
     xboxController.povUp().whileTrue(new RunUptake(m_uptake, 0.5));
     xboxController.y().whileTrue(new AmpFlipperForward(m_flipper));
+
+    xboxController
+        .x()
+        .whileTrue(
+            new ShootNStrafe(
+                m_swerveDrive,
+                m_vision,
+                () -> leftJoystick.getRawAxis(1),
+                () -> leftJoystick.getRawAxis(0),
+                () -> rightJoystick.getRawAxis(0),
+                600));
```

Comment:
Why 600 specifically?

Reply:
placeholder (other than 420.69 for reasons)